src/AppBridgeBlock.ts

```diff
@@ -423,7 +418,7 @@ export class AppBridgeBlock {
         };
     }
 
-    private mapAssetChooserResultToFrontifyAsset(asset: AssetChooserResult): FrontifyAsset {
+    private mapAssetChooserResultToFrontifyAsset(asset: AssetChooserResult): Asset {
```

Comment:
We should update the name of the method as well:
```suggestion
    private mapAssetChooserResultToAsset(asset: AssetChooserResult): Asset {
```